app/views/admin/job_applications/_job_application_head.html.haml

```diff
@@ -32,12 +32,15 @@
         = link_to t('.nav_files', count: 0), "#files", class: "nav-link", data: {toggle: :tab}, role: :tab, aria: {controls: :files, selected: false}
   .col-12.col-md-4{class: job_application_modal_section_classes}
     .btn-group.my-0
-      = link_to job_application.aasm.human_state, "#", class: "btn btn-primary btn-raised dropdown-toggle", data: {toggle: :dropdown}, aria: {haspopup: true, expanded: false}
-      .dropdown-menu
-        - job_application.aasm.states.map(&:name).each do |state|
-          - klasses = %w(dropdown-item)
-          - klasses << 'disabled' if job_application.state.to_sym == state
-          = link_to JobApplication.human_attribute_name("state/#{ state }"), [:change_state, :admin, job_application, {state: state}], method: :patch, remote: true, class: klasses
+      - if can? :change_state, JobApplication
```

Comment:
be more precise:
```
if can? :manage, job_application
```

Reply:
c'est ce que j'avais mit au départ, mais je voulais être plus précis en précisant le contrôler.